Examples/Example9.SendPacket/Example09.SendPacket.csproj

```diff
@@ -97,6 +97,10 @@
       <Project>{24262E52-1304-4A25-8F73-A3B06E40592E}</Project>
       <Name>SharpPcap</Name>
     </ProjectReference>
+     <Reference Include="PacketDotNet, Version=0.10.0.0, Culture=neutral, PublicKeyToken=null">

```

Comment:
Didn't an earlier commit change the version to 0.13.0 from 0.10.0? But this one is 0.10.0?